src/qibolab/platforms/platform.py

```diff
@@ -81,6 +81,8 @@ def get_gain(self, qubit):
         raise_error(NotImplementedError, f"{self.name} does not support gain.")
 
     def set_bias(self, qubit, bias):
+        if self.qubits[qubit].flux is None:
+            raise_error(NotImplementedError, f"{self.name} does not have flux.")
```

Comment:
```suggestion
            raise_error(ValueError, f"{self.name} does not have flux.")
```

From the point of view of this function, there is nothing that is `NotImplemented`, since the missing implementation is happening before.

(baseline: think locally, do not couple the behavior of different functions, not even conceptually)